internal/lsp/diagnostics.go

```diff
@@ -34,11 +35,21 @@ func (s *Server) Diagnose(uris ...protocol.DocumentURI) {
 	defer s.syncDiagnostics()
 
 	// TODO(5nord): Run linter against uris
+	for _, uri := range uris {
+		s.client.PublishDiagnostics(context.TODO(), &protocol.PublishDiagnosticsParams{
+			Diagnostics: make([]protocol.Diagnostic, 0),
+			URI:         uri,
+		})
```

Comment:
This clears only currently changed files. Clearing the diagnostic should probably be done before overwriting s.diags. Like so for example:
```
for uri, _ := range s.diags {
    s.client.PublishDiagnostics(...)
}